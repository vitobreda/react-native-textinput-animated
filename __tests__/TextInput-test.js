import 'react-native';
import React from 'react';
import {TextinputAnimated} from '../lib';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
 const tree = renderer.create(<TextinputAnimated />);

 expect(tree).toMatchSnapshot();

});
