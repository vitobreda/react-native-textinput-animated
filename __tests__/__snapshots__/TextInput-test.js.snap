// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "marginBottom": 10,
        "paddingTop": 18,
        "width": "100%",
      },
      undefined,
    ]
  }
>
  <Text
    style={
      Object {
        "backgroundColor": undefined,
        "color": "rgba(170, 184, 194, 1)",
        "fontSize": 16,
        "left": 10,
        "position": "absolute",
        "top": 35,
        "zIndex": 2,
      }
    }
  />
  <TextInput
    allowFontScaling={true}
    autoCapitalize="none"
    onBlur={[Function]}
    onEndEditing={[Function]}
    onFocus={[Function]}
    rejectResponderTermination={true}
    style={
      Array [
        Object {
          "fontSize": 16,
          "height": 48,
          "paddingLeft": 15,
          "paddingTop": 10,
        },
        Object {
          "borderBottomColor": "#555",
          "borderBottomWidth": 1,
        },
      ]
    }
    underlineColorAndroid="transparent"
  />
</View>
`;
